packages/compass-crud/src/components/insert-document-dialog.less

```diff
@@ -38,114 +38,115 @@
   }
 }
 
-.document {
-  .document-elements {
-    width: 100%;
-
-    .editable-element-actions {
-      display: inline-block;
-      opacity: 0;
-      width: 18px;
-      line-height: 11px;
-      margin-right: 35px;
-      text-align: center;
-      cursor: pointer;
-      color: #807f7f;
-    }
-
-    .editable-element-field {
-      max-width: 10vw;
-    }
-
-    .editable-element-value {
-      max-width: 20vw; 
-    }
-
-    .editable-element-is-added, .editable-expandable-element-header-is-added {
-      background-color: #FFFFFF;
-
-      input {
-        background-color: #FFFFFF;
-
-        &:focus {
-          background-color: #FFFFFF;
-        }
-      }
-
-      .line-number {
-        background-color: #FFFFFF;
+.insert-document-dialog {
+  .document {
+    .document-elements {
+      width: 100%;
+
+      .editable-element-actions {
+        display: inline-block;
+        opacity: 0;
+        width: 18px;
+        line-height: 11px;
+        margin-right: 35px;
+        text-align: center;
+        cursor: pointer;
         color: #807f7f;
       }
-    }
 
-    .editable-element {
+      .editable-element-field {
+        max-width: 10vw;
+      }
 
-      &:hover {
-        background-color: #f5f6f7;
+      .editable-element-value {
+        max-width: 20vw;
+      }
 
-        &::before {
-          background-color: #f5f6f7;
-        }
+      .editable-element-is-added,
+      .editable-expandable-element-header-is-added {
+        background-color: #ffffff;
 
         input {
-          background-color: #f5f6f7;
+          background-color: #ffffff;
 
           &:focus {
-            background-color: #FFFFFF;
+            background-color: #ffffff;
           }
         }
 
-        .editable-element-actions {
-          opacity: 1;
+        .line-number {
+          background-color: #ffffff;
+          color: #807f7f;
         }
+      }
 
-        .editable-element-types {
-          .btn {
-            background-color: #dee0e3;
-            color: #313030;
-          }
-          .caret {
-            visibility: visible;
+      .editable-element {
+        &:hover {
+          background-color: #f5f6f7;
```

Comment:
I think we're losing the hover over field effect we used to have - is that an intentional change?
Before 
<img width="500" alt="Screen Shot 2022-02-09 at 1 44 38 PM" src="https://user-images.githubusercontent.com/1791149/153268764-0eaa78e2-aad3-416d-81a5-21a8803944b4.png">

After
<img width="410" alt="Screen Shot 2022-02-09 at 1 44 44 PM" src="https://user-images.githubusercontent.com/1791149/153268773-d055fba7-bae9-4d5a-8e41-06db03d2af92.png">

Reply:
Sure thing, whichever works, mostly interested in preserving the existing functionality if possible.